docs/api-gateway/08-01-expose-service.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Expose a service
+type: Tutorials
+---
+
+This tutorial shows how to expose services using the API Gateway Controller for the `GET` and `POST` methods.
```

Comment:
Maybe add sth that will indicate that we expose service endpoints and configure different http methods  for them